docs/README.md

```diff
@@ -100,46 +117,102 @@ or on Linux with
 with Chromium:
 
 ```bash
 chromium-browser docs/target/html/index.html
 ```
 
-Alternatively, you can start a web server with that folder as root, e.g. again
-with Python and then open [http://localhost:4000](http://localhost:4000) in a
-web browser.
+Alternatively, you can start a web server with that folder as root, such as with
+the following Python command. You can then open
+[http://localhost:4000](http://localhost:4000) in a web browser.
 
 ```bash
 cd docs/target/html/
 python3 -m http.server 4000
 ```
 
 In order to see any changes from the source files in the HTML output, simply
-re-run the make command and refresh the browser.
+re-run the ``build`` command and refresh the browser.
 
 ## Versioning
 
-The version displayed in the resulting HTML is read from the top level Maven
-`pom.xml` file `version` field, by default.
+The version displayed in the resulting HTML is read by default from the top level Maven
+`pom.xml` file `version` field.
 
-To deploy a specific documentation set (e.g. a SNAPSHOT version) as release
-version you have to override the pom version with the `TRINO_VERSION`
+To deploy a specific documentation set (such as a SNAPSHOT version) as the release
+version you must override the pom version with the `TRINO_VERSION`
 environment variable.
 
 ```bash
-TRINO_VERSION=327 docs/build
+TRINO_VERSION=355 docs/build
 ```
 
 If you work on the docs for more than one invocation, you can export the
-variable and use it with sphinx.
+variable and use it with Sphinx.
 
 ```bash
-export TRINO_VERSION=327
+export TRINO_VERSION=354
 docs/build
 ```
 
 This is especially useful when deploying doc patches for a release where the
 Maven pom has already moved to the next SNAPSHOT version.
 
+## Contribution requirements
+
+
+To contribute corrections or new explanations to the Trino documentation requires
+only a willingness to help and submission of your [Contributor License
+Agreement](https://github.com/trinodb/cla) (CLA).
+
+## Workflow
+
+The procedure to add a documentation contribution is the same as for [a code
+contribution](https://trino.io/development/process.html).
+
+* In the Trino project's [GitHub Issues
+  list](https://github.com/trinodb/trino/issues), identify documentation issues
+  by filtering on the [``docs``
+  label](https://github.com/trinodb/trino/issues?q=is%3Aissue+is%3Aopen+label%3Adocs).
+
+* If you want to help Trino documentation, but don't know where to start, look
+  in the Issues list for both the [``docs`` and ``good first issue``
+  labels](https://github.com/trinodb/trino/issues?q=is%3Aissue+is%3Aopen+label%3Adocs+label%3A%22good+first+issue%22).
+
+* If the doc fix you have in mind does not yet have an issue, add one (which
+  requires a signed CLA). Add the ``docs`` label to your new issue.
+
+* Discuss a small change in the #docs channel of the [Trino
+  Slack](https://trino.io/slack.html).
+
+* For a larger contribution, create a GitHub pull request as described
```

Comment:
This is a little misleading, as any change large or small will eventually need a PR.

Reply:
Fixed.